flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -516,6 +527,30 @@ public void registerType(Class<?> type) {
 	}
 
 	// --------------------------------------------------------------------------------------------
+	//  Time characteristic
+	// --------------------------------------------------------------------------------------------
+
+	/**
+	 * Sets the time characteristic for the stream, e.g., processing time, event time,
+	 * or ingestion time.
+	 * 
+	 * @param characteristic The time characteristic.
+	 */
+	public void setStreamTimeCharacteristic(TimeCharacteristic characteristic) {
```

Comment:
Are there plans to allow setting different time characteristics for different streams in the same topology?
For example an ingestion time and a event time stream?